osu.Game/Screens/Play/SongProgressInfo.cs

```diff
@@ -76,18 +76,18 @@ protected override void Update()
             double songCurrentTime = AudioClock.CurrentTime - startTime;
             int currentPercent = Math.Max(0, Math.Min(100, (int)(songCurrentTime / songLength * 100)));
             int currentSecond = (int)Math.Floor(songCurrentTime / 1000.0);
-
             if (currentPercent != previousPercent)
             {
                 progress.Text = currentPercent.ToString() + @"%";
                 previousPercent = currentPercent;
             }
-
             if (currentSecond != previousSecond && songCurrentTime < songLength)
             {
-                timeCurrent.Text = TimeSpan.FromSeconds(currentSecond).ToString(songCurrentTime < 0 ? @"\-m\:ss" : @"m\:ss");
-                timeLeft.Text = TimeSpan.FromMilliseconds(endTime - AudioClock.CurrentTime).ToString(@"\-m\:ss");
-
+                //since TimeSpan to string doesnt support minutes above 60, we have to do something else.
+                var timeRemain = endTime - AudioClock.CurrentTime;
+                string minutes = songCurrentTime < 0 ? "-0" : Math.Floor(TimeSpan.FromSeconds(currentSecond).TotalMinutes).ToString("");
+                timeCurrent.Text = minutes + ':' + TimeSpan.FromSeconds(currentSecond).ToString(@"ss");
+                timeLeft.Text = "-" + Math.Floor(TimeSpan.FromMilliseconds(timeRemain).TotalMinutes) + ':' + TimeSpan.FromMilliseconds(timeRemain).ToString(@"ss");
```

Comment:
Extract this into a method, and the following is a less convoluted way:

`private string formatTime(TimeSpan timeSpan) => $"{(timeSpan < TimeSpan.Zero ? "-" : "")}{timeSpan.Duration().TotalMinutes:N0}:{timeSpan.Duration().Seconds:D2}";`

with tests:

```
[Test]
public void TestNegativeTime()
{
    Assert.AreEqual("-0:01", formatTime(TimeSpan.FromMilliseconds(-1000)));
    Assert.AreEqual("-0:10", formatTime(TimeSpan.FromMilliseconds(-10000)));
    Assert.AreEqual("-1:00", formatTime(TimeSpan.FromMilliseconds(-60000)));
    Assert.AreEqual("-1:05", formatTime(TimeSpan.FromMilliseconds(-65000)));
    Assert.AreEqual("-65:00", formatTime(TimeSpan.FromMilliseconds(-3900000)));
    Assert.AreEqual("-65:05", formatTime(TimeSpan.FromMilliseconds(-3905000)));
}

[Test]
public void TestPositive()
{
    Assert.AreEqual("0:01", formatTime(TimeSpan.FromMilliseconds(1000)));
    Assert.AreEqual("0:10", formatTime(TimeSpan.FromMilliseconds(10000)));
    Assert.AreEqual("1:00", formatTime(TimeSpan.FromMilliseconds(60000)));
    Assert.AreEqual("1:05", formatTime(TimeSpan.FromMilliseconds(65000)));
    Assert.AreEqual("65:00", formatTime(TimeSpan.FromMilliseconds(3900000)));
    Assert.AreEqual("65:05", formatTime(TimeSpan.FromMilliseconds(3905000)));
}

[Test]
public void TestFractional()
{
    Assert.AreEqual("0:01", formatTime(TimeSpan.FromMilliseconds(1234)));
    Assert.AreEqual("0:12", formatTime(TimeSpan.FromMilliseconds(12345)));
    Assert.AreEqual("1:05", formatTime(TimeSpan.FromMilliseconds(65432)));
}

[Test]
public void TestZero()
{
    Assert.AreEqual("0:00", formatTime(TimeSpan.Zero));
}
```